PSKoans/Koans/Foundations/AboutArrays.Koans.ps1

```diff
@@ -44,6 +46,53 @@ Describe 'Arrays' {
         __ | Should -Be $Names[4]
     }
 
+    It 'arrays are fixed size, elements cannot be added or removed' {
+        # Most arrays in PowerShell are fixed size. Elements cannot be directly added or removed.
+
+        $Ages = 12, 25, 18, 64
+
+        { $Ages.Add(59) } | Should -Throw -ExpectedMessage 'Collection was of a fixed size.'
+        { $Ages.Remove(12) } | Should -Throw -ExpectedMessage 'Collection was of a fixed size.'
+    }
+
+    It 'new arrays are created using the addition operator' {
+        <#
+            To add an element to a fixed size array, a new array must be created. PowerShell does this
+            in the background when the addition operator is used.
+        #>
+
+        $Age = __
+        $Ages = $Ages + $Age
```

Comment:
I think you may possibly have issues here with how Pester scopes things.... 

Might be best if even just for clarity to re-declare the `$Ages` array at the top of the `It`